assessments/array-assessment.js

```diff
@@ -15,24 +15,25 @@ arrayOfStrings.forEach(item => console.log(item));
 // 2.
 // For arrayOfNumbers, return a new array where the values are multiplied by 10
 // expected output: [50, 40, 30...]
-
+arrayOfNumbers.forEach(mult => console.log(mult * 10));
```

Comment:
use `.map` here, because you want to return a new array, where each value gets X 10. Map will always do the same operation for all items in the array, essentially you 'map' the new values to the old.

```
arrayOfNumbers.map(number => number * 10)
```